packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -39,6 +39,11 @@ model JsonArrayPartsRequest {
   previousAddresses: Address[];
 }
 
+model PureMultiBinaryPartsRequest {
+  profileImage: bytes;
+  pictures?: bytes;
```

Comment:
nit, singular form as this is only 1 file?

Reply:
I want to reuse [checkPngFile](https://github.com/Azure/cadl-ranch/blob/82d302643fac9c1192a9ac7ecdfb308e4b4cebed/packages/cadl-ranch-specs/http/payload/multipart/mockapi.ts#L35-L38)

Reply:
You can have it `function checkPngFile(req: MockRequest, fieldName: string), file: Record<string, any>`

but it's up to you...